packages/charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -58,11 +53,16 @@ export class Arc extends React.Component<IArcProps, { isCalloutVisible: boolean
     }
   }
 
-  private _hoverOff(): void {
+  private _hoverOff = (): void => {
     this.props.hoverLeaveCallback!();
-  }
+  };
 
   private _redirectToUrl(href: string | undefined): void {
     href ? (window.location.href = href) : '';
   }
 }
+
+function _updateChart(newProps: IArcProps): void {
+  newProps.arc.innerRadius(newProps.innerRadius);
+  newProps.arc.outerRadius(newProps.outerRadius);
```

Comment:
I had left a previous comment, but misread the code. This is just calling a function, not mutating props.